tests/metadata-zip.spec.ts

```diff
@@ -78,14 +78,18 @@ describe('metadata zip', function() {
 	const blockMap = parse(
 		outdent`
 			<?xml version="1.0" ?>
-			<bmap version="1.3">
+			<bmap version="2.0">
```

Comment:
Is it worth creating separate test files, to check that **both** bmap versions are correctly supported? Or does that lie outside the scope of etcher-sdk? :man_shrugging:

Reply:
I thought about it, but 2.0 has been around since 2013.